aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -216,7 +216,7 @@ Use <xref:Microsoft.AspNetCore.Components.NavigationManager> to manage URIs and
 | Member | Description |
 | ------ | ----------- |
 | <xref:Microsoft.AspNetCore.Components.NavigationManager.Uri> | Gets the current absolute URI. |
-| <xref:Microsoft.AspNetCore.Components.NavigationManager.BaseUri> | Gets the base URI (with a trailing slash) that can be prepended to relative URI paths to produce an absolute URI. Typically, <xref:Microsoft.AspNetCore.Components.NavigationManager.BaseUri> corresponds to the `href` attribute on the document's `<base>` element in `wwwroot/index.html` (Blazor WebAssembly) or `Pages/_Layout.cshtml` (Blazor Server). |
+| <xref:Microsoft.AspNetCore.Components.NavigationManager.BaseUri> | Gets the base URI (with a trailing slash) that can be prepended to relative URI paths to produce an absolute URI. Typically, <xref:Microsoft.AspNetCore.Components.NavigationManager.BaseUri> corresponds to either the `href` attribute on the document's `<base>` element in `wwwroot/index.html` (Blazor WebAssembly) or `Pages/_Layout.cshtml` (Blazor Server), or the `PathBase` property of the `HttpRequest` when pre-rendering. |
```

Comment:
This is not really true unless you use `~/` in your base tag

Reply:
What is not really true, regarding the addition I made there?

When pre-rendering the NavigationManager is initialized from the current request, no blazor hub / `<base />` involved.  See `StaticComponentRenderer` initializing the `NavigationManager` from the `request` [`GetContextBaseUri(httpContext.Request)`](https://github.com/dotnet/aspnetcore/blob/ae1a6cbe225b99c0bf38b7e31bf60cb653b73a52/src/Mvc/Mvc.ViewFeatures/src/RazorComponents/StaticComponentRenderer.cs#L80). It passes the request to `GetContextBaseUri` which builds the absolute URI using only `Scheme` + `Host` + `PathBase`. 

So the wording should be along the lines of "or the absolute URI to the `PathBase` property of the `HttpRequest` when pre-rendering.".

Reply:
... and btw while you sort that out, I say again that this added bit pertains to one hosting model (apparently). We need to mention it if it's the case. I'll work on the grammar/style bits once it's clarified.